src/app/content/components/Toolbar/PracticeQuestionsButton.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import { useSelector } from 'react-redux';
+import styled from 'styled-components/macro';
+import practiceQuestionsIcon from '../../../../assets/practiceQuestionsIcon.svg';
+import { practiceQuestionsEnabled } from '../../practiceQuestions/selectors';
+import { PlainButton, toolbarDefaultButton, toolbarDefaultText } from './styled';
+
+// tslint:disable-next-line:variable-name
+export const PracticeQuestionsWrapper = styled(PlainButton)`
+  ${toolbarDefaultButton}
+`;
+
+// tslint:disable-next-line:variable-name
+const PracticeQuestionsIcon = styled.img`
+  padding: 0.2rem;
```

Comment:
why not ` ${toolbarIconStyles}`?

Reply:
icon looks bigger than in the design with these styles